developers/weaviate/concepts/indexing/vector-indexes.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Vector indexes overview
+sidebar_position: 10
+image: og/docs/indexing.jpg
+# tags: ['vector index plugins']
+---
+
+Weaviate is a vector database. Most objects in Weaviate collections have one or more vectors. Individual vectors can have thousands of dimensions. Collections can have millions of objects. The resulting vector space can be exceedingly large.
```

Comment:
I believe the vector space size is independent of the number of vectors. A vector space is the precision of each dimension x number of dimensions. In this case floats x n_dimensions. The number of actual vectors in the DB does not change the size of the vector space.